docs/dg/dev/development-getting-started-guide.md

```diff
@@ -16,25 +16,67 @@ This document helps you get started with the Spryker Cloud Commerce OS. It has b
 ## 1. Install Spryker
 
 Spryker Demo Shops are a good starting point for any project. They are shipped with different sets of components, which are specific to respective business models. Demo Shops are fully functional and can be used for both demonstrative purposes as well as as a boilerplate for a new project. Though each shop comes with pre-selected components, Spryker offers hundreds of additional modules which you can add late.
-
 You can choose from the following options:
-
 - [B2C Demo Shop](/docs/about/all/b2c-suite.html)
 - [B2B Demo Shop](/docs/about/all/b2b-suite.html)
 - [Marketplace B2C Demo Shop](/docs/about/all/spryker-marketplace/marketplace-b2c-suite.html)
 - [Marketplace B2B Demo Shop](/docs/about/all/spryker-marketplace/marketplace-b2b-suite.html)
 
 You can run Spryker on MacOS, Linux, and Windows with WSL1 or WSL2. For installation instructions, see [Set up Spryker locally](/docs/dg/dev/set-up-spryker-locally/set-up-spryker-locally.html).
 
+## 2. Configure development tools
+
+Spryker offers a set of [development tools](docs/dg/dev/development-tools.html) that make it easier to work with the project: DockerSdk, Xdebug, WebProfiler, Code Quality Tools, and more.
+
+## 3. Configure repository and Continuous Integration
+
+To configure the repository and Continuous Integration, see [Configure CI](/docs/dg/dev/ci.html).
+
+## 4. Configure PhpStorm
+
+When you start developing a project, you need to reset it quite often. PhpStorm indexing can slow it down.
+To disable cache indexing, in the PhpStorm, right-click the folder and select **Mark Directory As&nbsp;<span aria-label="and then">></span> Excluded**.
+It is safe to disable cache indexing for the following files:
+- `data/cache`
+- `data/tmp`
+- `public/(Yves/Zed/Marketlace)/assets`
+- `.angular/cache`
+
+A couple of plugins for PhpStorm from community are recommended:
+![phpstorm plugins](https://spryker.s3.eu-central-1.amazonaws.com/docs/dg/dev/spryker-phpstorm-plugins.png)
+
+## 5. Explore Spryker Documentation
+
+To learn about Spryker architecture, different parts of the Client, Shared, Zed, and Yves folders, and their different layers, see the following documents:
+- [Architecture](/docs/dg/dev/architecture/architecture.html).
+- [Guidelines](/docs/dg/dev/guidelines/guidelines.html).
+- [Backend development](/docs/dg/dev/backend-development/back-end-development.html)
+- [Frontend development](/docs/dg/dev/frontend-development/latest/frontend-development.html)
```

Comment:
i would add - keep your project good for the future upgraide - https://docs.spryker.com/docs/dg/dev/sdks/sdk/customization-strategies-and-upgradability